toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -431,10 +437,15 @@ func (b *LiveOSIsoBuilder) updateGrubCfg(isoGrubCfgFileName string, pxeGrubCfgFi
 		return fmt.Errorf("failed to update the root kernel argument in the iso grub.cfg:\n%w", err)
 	}
 
-	inputContentString, err = updateSELinuxCommandLineHelperAll(inputContentString, imagecustomizerapi.SELinuxModeDisabled,
-		true /*allowMultiple*/, false /*requireKernelOpts*/)
+	err = verifyLiveOsSelinuxSupport(savedConfigs.OS.DracutPackageInfo, savedConfigs.OS.SELinuxPolicyPackageInfo)
 	if err != nil {
-		return fmt.Errorf("failed to set SELinux mode:\n%w", err)
+		logger.Log.Warnf("SELinux cannot be enabled due to older dracut and selinux-policy package versions.\n%w", err)
```

Comment:
If the versions are too old and the base image has SELinux enabled, then I think it is ok to only report a warning. But if the user explicitly asks for SELinux, then I think we should error out.

Reply:
Updated the logic. I did not take into account a user turning on selinux from the kernel command line arguments directly assuming that they should use `os.selinux.mode`. I think we do not have to cover this case - especially since it's a transient state until the updated packages are in all base images.